src/components/AutoUpdateTime.tsx

```diff
@@ -2,33 +2,26 @@
  * Displays the user's local time and updates it every minute.
  * The time auto-update logic is extracted to this component to avoid re-rendering a more complex component, e.g. DetailsPage.
  */
-import PropTypes from 'prop-types';
 import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
 import {View} from 'react-native';
 import DateUtils from '@libs/DateUtils';
 import useThemeStyles from '@styles/useThemeStyles';
+import {Timezone} from '@src/types/onyx/PersonalDetails';
 import Text from './Text';
-import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import withLocalize, {WithLocalizeProps} from './withLocalize';
 
-const propTypes = {
+type AutoUpdateTimeProps = {
     /** Timezone of the user from their personal details */
-    timezone: PropTypes.shape({
-        /** Value of selected timezone */
-        selected: PropTypes.string,
+    timezone: Required<Timezone>;
```

Comment:
```suggestion
    timezone: Timezone;
```
The object is required but properties inside can be undefined ☝️

Reply:
Done :)